pages/linux/phpenmod.md

```diff
@@ -1,6 +1,7 @@
 # phpenmod
 
 > Enable PHP extensions on Debian-based OSes.
+> More information: https://linuxcommandlibrary.com/man/phpenmod
```

Comment:
I think it's better to link the source code in this case.
```suggestion
> More information: <https://salsa.debian.org/php-team/php-defaults>.
```

Reply:
Ok, thank you.
With the code source, the user has to browse and find the accurate part. Is it still better because it's the source of this command, in this case?

Reply:
same discussion applies here? https://github.com/tldr-pages/tldr/pull/7323#discussion_r741107503

Reply:
Yes!